screens/Profile1/Profile.js

```diff
@@ -156,6 +160,24 @@ class Contact extends Component {
       console.log(result); //check output
       if (!result.cancelled) {
         this.setState({ image: result.uri });
+        fetch('http://'+global.api+'/user/10101/profilepic/', {
+        method: 'PUT',
+        headers: {
+          Accept: 'application/json',
+          'Content-Type': 'application/json',
+          },
+          body: JSON.stringify({
+                //create unique match id, can get number of row in db + 1
+                "profilepic": JSON.stringify(result.uri)
```

Comment:
The profile image uri is the filepath relative to the phone's device. This wouldn't work.